modules/candidate_parameters/jsx/DiagnosisEvolution.js

```diff
@@ -0,0 +1,223 @@
+          record.TrajectoryName,
+          record.Project,
+          record.OrderNumber,
+          record.visitLabel,
+          record.instrumentName,
+          sourceFields,
+          formattedDiagnosis,
+          confirmed,
+          record.LastUpdate,
+        ]
+      );
+    });
+    return formattedDxEvolution;
+  }
+
+  /**
+   * Render latest diagnosis element
+   * @param {*} latestDiagnosis
+   * @return {JSX} - React markup for the component
+   */
+  renderLatestDiagnosis(latestDiagnosis) {
+    let element = [];
+
+    latestDiagnosis.map((entry) => {
+      const projectName = this.state.data.projects[entry.ProjectID];
+      let diagnosis = [];
+      Object.entries(JSON.parse(entry.Diagnosis)).map((entry) => {
+        const [fieldName, dx] = entry;
+        diagnosis.push(
+          <StaticElement
+            key={fieldName}
+            label={fieldName}
+            text={dx}
+          />
+        );
+      });
+
+      element.push(
+        <FieldsetElement
+          key={entry.DxEvolutionID}
+          legend={<h5>{projectName} - {entry.Name}</h5>}
+          class='col-md-6'
+        >
+          {diagnosis}
+        </FieldsetElement>
+      );
+    });
+    return element;
+  }
+
+  /**
+   * Renders the React component.
+   *
+   * @return {JSX} - React markup for the component
+   */
+  render() {
+    if (this.state.error) {
+        return <h3>An error occured while loading the page.</h3>;
+    }
+
+    if (!this.state.isLoaded) {
+        return <Loader/>;
+    }
+
+    const latestDiagnosis = this.state.data.latestProjectDiagnosis.length > 0 ?
+      <div className='col-md-10'>
+        <h3>Latest Diagnosis</h3>
+        <p>This diagnosis is <strong style={{color: 'red'}}>
+          unconfirmed</strong>.
+          A confirmed diagnosis is one that belongs to an approved visit.
+        </p>
+        {this.renderLatestDiagnosis(this.state.data.latestProjectDiagnosis)}
+      </div>
+      : null;
+    const latestConfirmedDiagnosis =
+      this.state.data.latestConfirmedProjectDiagnosis.length > 0 ?
+        <div className='col-md-10'>
+          <h3>Latest Confirmed Diagnosis</h3>
+          <p>This diagnosis is <strong style={{color: 'green'}}>
+            confirmed</strong>.
+            A confirmed diagnosis is one that belongs to an approved visit.
+          </p>
+          {this.renderLatestDiagnosis(
+            this.state.data.latestConfirmedProjectDiagnosis
+          )}
+        </div>
+        : null;
+
+    return (
+      <div className='row'>
+        <FormElement
+          name='diagnosisEvolution'
+          onSubmit={this.handleSubmit}
+          ref='form'
+          class='col-md-12'
+        >
+          <StaticElement
+            label='PSCID'
+            text={this.state.data.pscid}
+          />
+          <StaticElement
+            label='DCCID'
+            text={this.state.data.candID}
+          />
+          {latestDiagnosis}
+          {latestConfirmedDiagnosis}
```

Comment:
```suggestion
          {latestConfirmedDiagnosis || latestDiagnosis}
```

at the moment, both confirmed and not confirmed show up. i assume that once the diagnosis is confirmed, that's the only one that should be shown?

![Screen Shot 2022-06-16 at 12 54 06 PM](https://user-images.githubusercontent.com/17415878/174125715-8e10623a-d7fc-4f98-99a9-cbae1f9b24d2.png)

Reply:
Not necessarily, there could be a confirmed diagnosis from V1 and an unconfirmed diagnosis from V2. In this case, the V2 unconfirmed dx is actually the latest.

However the case shown in this screenshot could call for a wording change. The unconfirmed section could say to be cautious that it MAY be unconfirmed ? What do you think ?

Reply:
hmm i see, that makes sense. i think if only 1 diagnosis trajectory exists though and its confirmed, then we should only be showing that its confirmed, since its a bit confusing if both confirmed and unconfirmed exist. could we do something like this?

<img width="1415" alt="Screen Shot 2022-06-16 at 6 16 12 PM" src="https://user-images.githubusercontent.com/17415878/174185836-9d438f30-ac3d-4dbd-a5cd-5e7366fb7113.png">
<img width="1417" alt="Screen Shot 2022-06-16 at 6 16 04 PM" src="https://user-images.githubusercontent.com/17415878/174185837-2ffcc8cc-e450-406b-8bde-72c577f622a5.png">

Reply:
ill request line changes in a separate comment for this to work

Reply:
yes I'm okay with this change